src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
+    
+      <fieldset class="group">
```

Comment:
```suggestion
    >
         <fieldset class="group">
```
Don't leave newlines between parent and child elements